cartography/intel/aws/cloudfront.py

```diff
@@ -70,12 +76,18 @@ def _load_cloudfront_distributions_tx(tx: neo4j.Transaction, distributions: List
         distribution.viewer_protocol_policy = record.DefaultCacheBehavior.ViewerProtocolPolicy,
         distribution.web_acl_id = record.WebACLId,
         distribution.is_ipv6_enabled = record.IsIPV6Enabled,
-        distribution.http_version = record.HttpVersion
-    WITH distribution
+        distribution.http_version = record.HttpVersion,
+        distribution.s3_bucket_name = record.S3bucketName
+    WITH distribution, record
     MATCH (owner:AWSAccount{id: $AWS_ACCOUNT_ID})
     MERGE (owner)-[r:RESOURCE]->(distribution)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = $aws_update_tag
+    WITH distribution, record
+    MATCH (bucket:S3Bucket {name: record.S3bucketName})
```

Comment:
do we have an index on name field for S3Bucket? if not, we shoudl create one. This will impact performance of the insert query.